profiles/package.mask

```diff
@@ -65,12 +65,15 @@ dev-java/ant-xz
 dev-java/gnu-jaf
 dev-java/failureaccess
 
+# Oz Tiram <oz.tiram@gmail.com> (2024-03-10)
+# plette in version 1.0.0 is broken and pipenv is still
+# not using it. Bug #925611
+=dev-python/plette-1.0.0
```

Comment:
You're missing empty line here. Also, I don't get the point of masking it if you pin to it anyway.

Reply:
OK. You are correct we can do with pinning plette to 0.4.4 only.